src/main/kotlin/lotto/domain/value/HitLotto.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.domain.value
+
+import java.math.BigInteger
+
+enum class HitLotto(private val index: Int, val money: Money, private var count: Int = 0) {
```

Comment:
다른 개발자도 예상할 수 있는 기능일까요? 아래의 글이 도움 되면 좋겠어요.

- https://woowabros.github.io/tools/2017/07/10/java-enum-uses.html
- https://kotlinlang.org/docs/reference/enum-classes.html

```suggestion
enum class HitLotto(private val index: Int, val money: Money) {
```